include/mouse.h

```diff
@@ -79,4 +119,105 @@ uint8_t MOUSEBIOS_GetSampleRate();
 void MOUSEDOS_BeforeNewVideoMode();
 void MOUSEDOS_AfterNewVideoMode(const bool setmode);
 
+// ***************************************************************************
+// MOUSECTL.COM / GUI configurator interface
+// ***************************************************************************
+
+class MouseInterface;
+class MousePhysical;
+
+class MouseInterfaceInfoEntry final {
+public:
+
+    bool IsEmulated() const;
+    bool IsMapped() const;
+    bool IsMapped(const uint8_t device_idx) const;
+    bool IsMappedDeviceDisconnected() const;
+
+    MouseInterfaceId GetInterfaceId() const;
+    MouseMapStatus GetMapStatus() const;
+    const std::string &GetMappedDeviceName() const;
+    uint8_t GetSensitivityX() const; // 1-99
+    uint8_t GetSensitivityY() const; // 1-99
```

Comment:
A powerful feature of DOSBox's current sensitivity is that 0 can be used to prevent input on that specific axis - for games like DOOM, so you can use "mouse-look" type actions to look left and right, and WASD keys for position sliding, with the assurance that any accident forward/back mouse movement won't "stall" your guy's movement.

Also, negative values invert the axis to help normalize those strange FPS games that unfortunately used flight-sim-type direction: "Pull back to look up", push forward to look down. :rofl: 

Adding @GranMinigun

Reply:
(note for myself) reviewed up to here